conf/cmi/automated_cron.settings.yml

```diff
@@ -1,4 +1,4 @@
 _core:
   default_config_hash: fUksROt4FfkAU9BV4hV2XvhTBSS2nTNrZS4U7S-tKrs
 langcode: fi
-interval: 10800
```

Comment:
This setting should be changed to "Never" in /admin/config/system/cron, because Drupal cron is manually ran every 10 minutes through drush in `drupal-employment-services/docker/openshift/crons/base.sh`